lib/peep.rb

```diff
@@ -0,0 +1,40 @@
+require 'pg'
+
+class Peep
+  attr_reader :content, :id, :time
+
+  def initialize(time: Time.now.to_s[0..-7], id: nil, content: nil)
```

Comment:
If the column in your database accepts a timestamp value, is it a good idea to change the type of the time variable to a string?